playbooks/openstack/templates/cluster-stack-template.yaml.j2

```diff
@@ -0,0 +1,27 @@
+apiVersion: clusterstack.x-k8s.io/v1alpha1
+kind: ClusterStack
+metadata:
+  name: clusterstack
+spec:
+  provider: openstack
+  name: alpha
+  kubernetesVersion: "1.29"
```

Comment:
Should this information be extracted from the cluster stack name?

Reply:
Yes, it should be possible. We already have `k8s_version` in `cluster.yaml.j2` template. But the output looks like `v1.29.3`. So we should add a new variable with a different name or is it possible to add there k8s version with the patch version also like `1.29.3`?

Reply:
https://github.com/SovereignCloudStack/cluster-stack-operator/blob/main/config/crd/bases/clusterstack.x-k8s.io_clusterstacks.yaml#L90

Reply:
see https://github.com/SovereignCloudStack/cluster-stacks/pull/53/commits/55eed4cca7a875ff181c452fc360a6bb54d9b379